app/controllers/reports_controller.rb

```diff
@@ -1,13 +1,23 @@
 # frozen_string_literal: true
 class ReportsController < ApplicationController
   def create
-    Report.create!(
-      user: current_user,
-      hospital: current_user.hospital,
-      informant_name: 'NÃO INFORMADO',
-      informant_function: 'NÃO INFORMADO'
+    service = ReportService.new(
+      informant_name: report_params[:informant_name],
+      informant_function: report_params[:informant_function],
+      user_id: current_user.id,
+      hospital_id: current_user.hospital.id
```

Comment:
```suggestion
      user: current_user,
      hospital: current_user.hospital
```

Não seria melhor passar o current_user e o hospital aqui? Nós temos esses objetos aqui, e ao passar apenas o id precisamos fazer um find novamente.